### PR TITLE
Update mouse event references

### PR DESCRIPTION
**Pull request by nickmcintyre**
Preparing references in `src/events/mouse.js` for the new website.

#### PR Checklist

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated

@Qianqianye @davepagurek @limzykenneth

### CODE REVIEW

**Comment by Qianqianye** [comment]
Thanks @nickmcintyre, could you double check the PR to make sure it passes the checks?

**Comment by nickmcintyre** [comment]
@Qianqianye they passed locally, not sure what's happening.

**Comment by limzykenneth** [comment]
I see you just fixed this as I was leaving the review comment. Will merge once the tests passed.

**Review by limzykenneth — APPROVED** [review]

File: src/events/mouse.js [hunk 1]
```diff
@@ -78,58 +107,245 @@ p5.prototype.movedY = 0;
...
+ * </code>
+ * </div>
+ *
+ * <div>
+ * <code>
+ * let font;
+ *
+ * // Load a font for WebGL mode.
+ * function preload() {
+ *   font = loadFont('assets/Inconsolata.otf');
```
[the remark this answers is not in the document]
Comment (nickmcintyre) [reply src/events/mouse.js]: 👍🏽

File: src/events/mouse.js [hunk 2]
```diff
@@ -78,58 +107,245 @@ p5.prototype.movedY = 0;
...
```
[the remark this answers is not in the document]
Comment (nickmcintyre) [reply src/events/mouse.js]: 👍🏽

**Pull request merged**